README.md

```diff
@@ -4,60 +4,114 @@ This repository contains examples showing how to consume the PowerDMS File Picke
+  - [Selected File Response](#Selected-File-Response)
 
-## How to integrate
+<br />
 
+## How to integrate
+At a high level, you do a few things to integrate the File Picker: 
 1. Add references to the initialization files to your page:
 
     ```html
     <link rel="stylesheet" href="https://filepicker.powerdms.com/initializer/powerDmsFilePicker.css" type="text/css">
 
     <script src="https://filepicker.powerdms.com/initializer/powerDmsFilePicker.js"></script>
     ```
+2. Prompt the user to authenticate.
 
-2. Add a button to the page that will call `window.PowerDms.initializePowerDmsFilePicker(config)`.
+3. Open the File Picker.
 
-3. Add a function that will call the PowerDMS API to get the file contents when a user makes a selection, and assign that to `config.onFileSelected` when the button is clicked.
+     Add a button to the page that will call `window.PowerDms.initializePowerDmsFilePicker(config)`.
 
-## Configuration
-`PowerDms` is a top level namespace that will house all PowerDms api calls. This includes the following calls:
+4. Handle the selection result.
 
+     Add a function that will call the PowerDMS API to get the file contents when a user makes a selection, and assign that to `config.onFileSelected` when the button is clicked.
 
-### `initializePowerDmsFilePicker`
-`initializePowerDmsFilePicker` accepts a configuration object as a single parameter. This object has the following structure:
+<br />
 
-#### JavaScript
 
-```javascript
-{
-  // String, required
-  // The API key given to your organization from PowerDMS.
-  apiKey,
+## Common Integration Scenarios
+The File Picker returns metadata about selected files. You may need to orchestrate additional API calls to achieve the intended integration experience. This section provides examples for some common scenarios.     
 
-  // Function, required
-  // A callback that is called when a user makes a selection.
-  onSelection,
+<br />
 
-  // Number, optional
-  // The desired width of the file picker, in pixels.
-  // Restricted from 566 to 1051.
-  width,
+### Downloading the Selected Document
+The File Picker selection object contains all of the information necessary to download the current revision of the selected file. To do so, call the `/documents/{documentId}/revisions/{revisionId}/content` endpoint using the `documentId` and `revisionId` from the selected file. For convenience, the correct link for the selected document is passed as the `contentUrl` property on the selection result. See the specific endpoint documentation (link tbd) for details.   
 
-  // Number, optional
-  // The desired height of the file picker, in pixels.
-  // Restricted from 350 o 650.
-  height
+<br />
+
+### Linking to the Selected Document
+You can construct a link to the selected document that allows an authenticated user to view the latest version of the selected document. The format of the link is `https://powerdms.com/link/{siteKey}/document/?id={documentId}` where:

```

Comment:
🔧 for a user that is not authenticated, it will ask the user to login to PowerDMS, and after authentication, it will redirect to the document

Reply:
I don't think that that detail is necessary for this documentation, which is how to construct that link from PowerDMS and File Picker API calls.